src/components/CreateArticles.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import * as selectors from '../selectors/createArticleSelector';
+import createArticleAction from '../actions/createArticles';
+import CreateArticle from '../views/CreateArticle';
+import CloudinaryWidget from './common/CloudinaryWidget';
+
+
+/**
+ * Form for creating a new article
+ * @param {object} props Component props
+ * @param {function} props.createArticleAction action creator function
+ */
+function CreateArticleContainer(props) {
+  /** passing global state loading as props into variables */
+  const {
+    message, isLoading, token,
```

Comment:
#### Nitpick
- Let's set authentication header with the token using a single instance of axios (Singleton pattern).

Reply:
feedback implemented